ush/check_ice_netcdf.sh

```diff
@@ -0,0 +1,38 @@
+#! /usr/bin/env bash
```

Comment:
@EricSinsky-NOAA 
I just noticed this script was added after an approval review was made on this PR and it slipped from being reviewed.
The function implemented here is the `datadep` with `age` that Rocoto already provides.
Besides, this is completely hardwired for GEFS options.
The function is checking the age of a file.
The file happens to be a ice netcdf file.
We will need to implement this via the proper rocoto means of checking if a file exists and if it has been present for 2 mins.